ivy/container/extensions.py

```diff
@@ -333,3 +333,105 @@ def lcm(
+        periodic: Optional[bool] = True,
+        dtype: Optional[Union[ivy.Dtype, ivy.NativeDtype]] = None,
+        *,        
+        key_chains: Optional[Union[List[str], Dict[str, str]]] = None,
+        to_apply: bool = True,
+        prune_unapplied: bool = False,
+        map_sequences: bool = False,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """
+        ivy.Container static method variant of ivy.hann_window. This method simply wraps 
+        the function, and so the docstring for ivy.hann_window also applies to this 
+        method with minimal changes.
+
+        Parameters
+        ----------
+        window_length
+            container including multiple window sizes.
+        periodic
+            If True, returns a window to be used as periodic function. 
+            If False, return a symmetric window.
+        dtype
+            The data type to produce. Must be a floating point type.
+        out
+            optional output container, for writing the result to.
+
+        Returns
+        -------
+        ret
+            The container that contains the Hann windows.
+
+        Examples
+        --------
+        With one :class:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=3, b=5)
+        >>> ivy.hann_window(x)
+        [{
+            a: ivy.array([0.0000, 0.7500, 0.7500])
+            b: ivy.array([0.0000, 0.3455, 0.9045, 0.9045, 0.3455]) 
+        }]
+        """
+        return ContainerBase.multi_map_in_static_method(
+            "hann_window",
+            window_length,
+            periodic,
+            dtype,
+            key_chains=key_chains,
+            to_apply=to_apply,
+            prune_unapplied=prune_unapplied,
+            map_sequences=map_sequences,
+            out=out,
+        )
+
+    def hann_window(
+        self: ivy.Container,
+        periodic: Optional[bool] = True,
+        dtype: Optional[Union[ivy.Dtype, ivy.NativeDtype]] = None,
+        *,
+        out: Optional[ivy.Container] = None,
+    ) -> ivy.Container:
+        """ivy.Container instance method variant of ivy.hann_window. This method simply
+        wraps the function, and so the docstring for ivy.hann_window also applies to 
+        this method with minimal changes.
+
+        Parameters
+        ----------
+        self
+            input container with window sizes.
+        periodic
+            If True, returns a window to be used as periodic function. 
+            If False, return a symmetric window.
+        dtype
+            The data type to produce. Must be a floating point type.
+        out
+            optional output container, for writing the result to.
+
+        Returns
+        -------
+        ret
+            The container containing the Hann windows.
+
+        Examples
+        --------
+        With one :class:`ivy.Container` input:
+
+        >>> x = ivy.Container(a=3, b=5)
+        >>> ivy.hann_window(x)
```

Comment:
should be an instance method call here.   about the square brackets in the result printed below (and above), not sure where they came from... ususally container is printed with just the curly brackets